profiles/base/package.use.mask

```diff
@@ -6,6 +6,10 @@
 # This file is only for generic masks. For arch-specific masks (i.e.
 # mask everywhere, unmask on arch/*) use arch/base.
 
+# Matt Turner <mattst88@gentoo.org> (2022-06-15)
+# GTK+2, known to be broken.
+dev-util/perf gtk
```

Comment:
Known to be broken where? I have this compiled and working. Here is proof:
![screenshot](https://user-images.githubusercontent.com/249404/174016202-182e852d-1d39-449d-9f20-8b8fccdc2cdc.png)
You may argue that this feature is not so useful, but it's definitely not broken.

Reply:
Sorry for the latency. Lots of other things occupying me...

See https://lore.kernel.org/lkml/20220415050003.3257645-1-mattst88@gmail.com/T/ where I resent a patch to port the tool to GTK+3.

> One problem I've noticed is that C++ symbol names are often broken.
> E.g.:
> 
> (perf:3254940): Gtk-WARNING **: 21:47:46.663: Failed to set text from
> markup due to error parsing markup: Error on line 1 char 95:
> “std::char_traits<char” is not a valid name: “<”
> 
> Presumably we need to call g_markup_escape_text() somewhere, but I
> struggled to find the right place.
> 
> Without the patch, the GTK-2 version repeatedly segfaults, so this is an
> improvement regardless.

Maybe the segfaulting issue was due to symbols with names containing `<`? I'll try to reproduce it, as well as resending the patch.

Reply:
Ah, nice work! I see what you mean now, I guess demangled symbols can be too much for GTK. I hope the port to GTK+3 makes it in before the next merge window. perf is a really nice tool, it's a pity that the graphical interface features are quite limited.